### ISSUE TITLE
getOnlineStatus not updating

### ISSUE THREAD

**Issue by Maxfojtik** [DV8FromTheWorld/JDA#488]
I don't know if i'm just not doing it right, but I can't seem to get this method to return any new data. This is my code so far.
```java
static public class EventList implements EventListener
	{
	    @Override
	    public void onEvent(Event event)
	    {
	    	if(event instanceof UserOnlineStatusUpdateEvent)
	    	{
	    		UserOnlineStatusUpdateEvent UOSUE = (UserOnlineStatusUpdateEvent) event;
	    		for(int i = 0; i < friendIDs.size(); i++)
	    		{
	    			if(friendIDs.get(i).equals(UOSUE.getUser().getId()))
	    			{
	    				System.out.println(UOSUE.getUser().getName()+" changed from "+UOSUE.getPreviousOnlineStatus());
	    				f.aSyncStatuses.put(UOSUE.getUser().getId(), jda.asClient().getFriendById(UOSUE.getUser().getId()).getOnlineStatus());
	    			}
	    		}
	    	}
	    }
	}
```
aSyncStatuses is a HashMap btw.
Thanks, Max Fojtik

**Comment by MinnDevelopment** [comment]
The relationships game and online status are currently not being updated by JDA. This will be addressed in #494

**Comment by Maxfojtik** [comment]
Sweet, Thanks
